### PR TITLE
filter api and new charts methods

### PR DESCRIPTION
**Pull request by piyushmani**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -140,6 +140,19 @@ app.get('/api/jobLogs/', (req, res) => {
...
+    var userName = req.query.userName;
+    var reportNameName = req.query.reportName;
+    var startDate = req.query.startDate;
+    var endDate = req.query.endDate;
+    var page = (+req.query.page);
+    var pageSize = (+req.query.pageSize);
+    jobs.filterJobs(userName, reportNameName, startDate, endDate, page, pageSize).then(function (result) {
+        res.send(result);
+    }, function (err) {
+        res.send(err);
```
Comment (johibkhan2) [inline app/app.js]: log this error before returning

File: app/jobs.js [hunk 2]
```diff
@@ -436,6 +438,91 @@ var job = {
         }
 
 
+    },
+    filterJobs: async function(userName, reportName, startDate, endDate, page, pageSize){
+
+        var reportWhereClause={}
+        var schedularWhereClause={}
+        if (userName){
```
Comment (johibkhan2) [inline app/jobs.js]: use ternary operator instead of if as it is much cleaner as compared to if

File: app/jobs.js [hunk 3]
```diff
@@ -436,6 +438,91 @@ var job = {
...
+            reportWhereClause.userid=userName;
+        }
+        if (reportName){
```
Comment (johibkhan2) [inline app/jobs.js]: in all places use ternary

File: app/app.js (hunk 1 above)
Comment (piyushmani) [reply app/app.js]: there is a logger already in jobs.js for this api.

**Pull request merged**